README.md

```diff
@@ -58,14 +69,21 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 
 Please check [CONTRIBUTING.md](./CONTRIBUTING.md) for more details about how to setup and develop the project.
 
+## Developer Documentation
```

Comment:
Reference it from CONTRIBUTING.md too?

Reply:
Addressed in https://github.com/microsoft/build-server-for-gradle/pull/189/commits/b624852d90f8acc723f4bb12128d95648ac18442